### DISCUSSION TITLE
app create from url and header dont work

### DISCUSSION THREAD

**Discussion by nrmn2492** [argoproj/argo-cd#8740] [Q&A]
Hello,
 I try to create an app from bitbucket repo file
`argocd app create node-dev -f https://api.bitbucket.org/2.0/repositories/asdas92/k8s-argocd/src/master/k8s-helm/nodejs/nodej/argocd-deploy-dev.yaml --header Authorization:Basic asdasdasd123123`

 ( https://api.bitbucket.org/2.0/repositories/asdas92/k8s-argocd/src/master/k8s-helm/nodejs/nodej/argocd-deploy-dev.yaml )
 if the repo is public, everything fine. app created, but if repo is private need auth to acces file (naturally)
 
`argocd app create node-dev -f https://api.bitbucket.org/2.0/repositories/asdas92/k8s-argocd/src/master/k8s-helm/nodejs/nodej/argocd-deploy-dev.yaml --header Authorization:Basic asdasdasd123123`
 
it does not work (empty terminal response), but if i try the curl command i get the file and ok.

`curl -H "Authorization:Basic asdasdasd123123" https://api.bitbucket.org/2.0/repositories/asdas92/k8s-argocd/src/master/k8s-helm/nodejs/nodej/argocd-deploy-dev.yaml` - Its ok.
 
what am i doing wrong, the "-H" parameter passes the header doesn't it?

Thanks

**Comment by Bogay** [comment]
Maybe you forgot the quotes? You only added them for `-H`.
```
argocd app create node-dev -f ... --header "Authorization:Basic asdasdasd123123"
```
